apps/docs/src/app/documentation/core-helpers/component-example/component-example.component.ts

```diff
@@ -1,5 +1,5 @@
 import { Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { RtlService } from '@fundamental-ngx/core/utils';
+import { DialogService, RtlService } from '@fundamental-ngx/core';
```

Comment:
Since @N1XUS implementing sub packaging, I think it's better to organize imports the following way:
```
import { DialogService } from '@fundamental-ngx/core/dialog';
import { RtlService } from '@fundamental-ngx/core/utils';
```

@InnaAtanasova @droshev @N1XUS Do you guys agree with that?